File: Week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int min_height = 1;
    int max_height = 8;
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < min_height || height > max_height);

    int space_counter = height - 1;
    int hash_counter = 1;

    for (int i = 1; i <= height; ++i)
    {
        for (int j = 1; j <= space_counter; ++j)
        {
            printf(" ");
        }

        for (int k = 1; k <= hash_counter; ++k)
        {
            printf("#");
        }

        printf("\n");

        space_counter--;
        hash_counter++;
    }
}

File: Week6/sentimental-mario-less/mario.py
from cs50 import get_int

min_height = 1
max_height = 8

while True:
    height = get_int("Height: ")

    if height in range(min_height, max_height + 1):
        break

for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)
